src/Extension/AccountReset/SecurityExtension.php

```diff
@@ -102,33 +100,23 @@ public function resetaccount(HTTPRequest $request)
 
     public function ResetAccountForm(): Form
     {
+        $field = ConfirmedPasswordField::create(
+            'Password',
+            _t(Member::class . '.NEWPASSWORD', 'New Password'),
+            titleConfirmField: _t(Member::class . '.CONFIRMNEWPASSWORD', 'Confirm New Password')
+        );
+        $field->setIsOnMemberForm(true);
         $fields = FieldList::create([
-            PasswordField::create(
-                'NewPassword1',
-                _t(
-                    'SilverStripe\\Security\\Member.NEWPASSWORD',
-                    'New password'
-                )
-            ),
-            PasswordField::create(
-                'NewPassword2',
-                _t(
-                    'SilverStripe\\Security\\Member.CONFIRMNEWPASSWORD',
-                    'Confirm new password'
-                )
-            ),
+            $field,
         ]);
-
         $actions = FieldList::create([
             FormAction::create('doResetAccount', 'Reset account'),
         ]);
-
-        $validation = RequiredFieldsValidator::create(['NewPassword1', 'NewPassword2']);
-
-        $form = Form::create($this->owner, 'ResetAccountForm', $fields, $actions, $validation);
-
+        // Not using RequiredFieldsValidator() here and instead "manually" checking that
+        // Password[_Password] and Password[_ConfirmPassword] are non-blank in doResetAccount()
+        // Done this way because ConfirmedPasswordField is not a standard CompositeField
```

Comment:
```suggestion
```
This comment isn't needed - `ConfirmedPasswordField` is handling the validation.

Reply:
Removed